blocks/init/src/blocks/components/hamburger/hamburger-style.scss

```diff
@@ -0,0 +1,78 @@
+/* stylelint-disable max-nesting-depth */
+
+$hamburger: (
+  colors: (
+    line: #2C2C2C,
+  ),
+  width: 24px,
+  line-height: 2px,
+  padding: 8px
+);
+
+body {
+  .hamburger {
+    @extend %button-reset;
+    $this: &;
+    background: transparent;
+    height: 100%;
+    padding: map-get-deep($hamburger, padding);
```

Comment:
no need for map-get-deep if there is only one level use map-get-strict (it is faster)